d2/src/main/proto/XdsD2.proto

```diff
@@ -40,5 +40,22 @@ message D2SymlinkNode {
 }
 
 message D2NodeMap {
+  // Deprecated in favor of D2UriMap
+  option deprecated = true;
   map<string, D2Node> nodes = 1;
 }
+
+message D2URI {
```

Comment:
please add comments to this new data structure.

Reply:
ACK

Reply:
Done